modules/slms_3/1.0/slms_3.smk

```diff
@@ -86,72 +87,94 @@ for tool in CFG_SLMS3["module_versions"].keys():
 include: "../../lofreq/" + CFG_SLMS3["module_versions"]["lofreq"] + "/lofreq.smk"
- 
+
 ##### FIRST PASS VARIANT CALLING RULES #####
 
 
 # Symlinks the input files into the module results directory (under '00-inputs/')
 
-rule _slms_3_input_strelka_vcf: 
+rule _slms_3_input_strelka_vcf:
     input:
-        vcf = str(rules._strelka_output_filtered_vcf.output.vcf), 
+        vcf = str(rules._strelka_output_filtered_vcf.output.vcf),
         tbi = str(rules._strelka_output_filtered_vcf.output.vcf_tbi)
     output:
-        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz", 
+        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz",
         tbi = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz.tbi"
     run:
         op.relative_symlink(input.vcf, output.vcf, in_module = True)
         op.relative_symlink(input.tbi, output.tbi, in_module = True)
 
-rule _slms_3_input_sage_vcf: 
+rule _slms_3_input_sage_vcf:
     input:
-        vcf = str(rules._sage_output_vcf.output.combined), 
+        vcf = str(rules._sage_output_vcf.output.combined),
         tbi = str(rules._sage_output_vcf.output.combined_tbi)
     output:
-        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.combined.vcf.gz", 
+        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.combined.vcf.gz",
         tbi = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.combined.vcf.gz.tbi"
     run:
         op.relative_symlink(input.vcf, output.vcf, in_module = True)
         op.relative_symlink(input.tbi, output.tbi, in_module = True)
 
-rule _slms_3_input_lofreq_vcf: 
+rule _slms_3_input_lofreq_vcf:
     input:
-        vcf = str(rules._lofreq_output_vcf.output.vcf_all), 
+        vcf = str(rules._lofreq_output_vcf.output.vcf_all),
     output:
-        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.lofreq.snvs.vcf.gz", 
+        vcf = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.lofreq.snvs.vcf.gz",
         tbi = CFG_SLMS3["dirs"]["inputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.lofreq.snvs.vcf.gz.tbi"
     run:
         op.relative_symlink(input.vcf, output.vcf, in_module = True)
         op.relative_symlink(input.vcf + ".tbi", output.tbi, in_module = True)
 
 # Annotate Strelka VCF and remove common GnomAD variants
+# bcftools annotate is very slow if one of the VCF files is quite large (see https://github.com/samtools/bcftools/issues/1199#issuecomment-624713745)
```

Comment:
Feel like this is not a problem when the cluster is used? Maybe we should drop this to avoid possible bugs in the future and avoid possible effect of snakefile modification on the samples already run through SLMS-3? This will also revert all modifications in SLMS-3 to master version

Reply:
As the entire dbSNP file is iterated through every time a VCF is annotated, this would definitely also be an issue on the GSC, perhaps even more so due to increased I/O bottlenecks. This probably hasn't been noticed as 1) The runtime of this step is relatively small when running exome or genome data through other pipeline steps, and 2) There are more variants, and over a wider range, compared to CAPPSeq data. This is most noticeable when running samples with only a handful (ex. 12) SNV calls for annotation, as this step is actually the bottleneck for both the CI and running capture data through

Reply:
I did test this to help ensure nothing is dropped, but I was unable to test this fully, as the annotation step also includes filtering steps. Will do one final comparison between the old version and new version